de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/externallyreferenced/InvoiceDetailItem.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * #%L
+ * de.metas.swat.base
+ * %%
+ * Copyright (C) 2019 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.invoicecandidate.externallyreferenced;
+
+import lombok.Builder;
+import lombok.Value;
+import org.apache.commons.lang3.StringUtils;
+
+@Value
+public class InvoiceDetailItem
+{
+	Integer seqNo;
+
+	String label;
+
+	String description;
+
+	@Builder
+	public InvoiceDetailItem(final Integer seqNo, final String label, final String description)
+	{
+		this.seqNo = seqNo;
+		this.label = label;
+		this.description = description;
+	}
+
+	/**
+	 * @return true if both label and description are blank, false otherwise.
+	 */
+	public boolean isEmpty()
+	{
+		return StringUtils.isBlank(label) && StringUtils.isBlank(description);
```

Comment:
Note: although we do have apache-StringUtils in the classpath, it's more common in metasfresh-code to use something like `de.metas.util.Check.isEmpty(label, true)`. 
(but for me it's ok like this)